src/main/java/com/hms/flexyosisoftconnector/configuration/OSIsoftConfig.java

```diff
@@ -198,6 +201,8 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
       httpTimeoutSeconds = HTTP_TIMEOUT_SECONDS_DEFAULT;
     }
 
+    autoRestart = appConfig.getBoolean("AutoRestart");

```

Comment:
add static final/constant for "AutoRestart"

Reply:
already static, and the final keyword would not work here with it being assigned in the initConfig function because blank static variable can only be assigned in constructors

Reply:
I mean for the actual string "AutoRestart", similar to how HTTP_TIMEOUT_SECONDS_KEY exists, in general the config file keys should be a constant.
```suggestion
    autoRestart = appConfig.getBoolean(AUTO_RESTART_KEY);
```

Reply:
fixed